qt_gui_cpp/package.xml

```diff
@@ -21,13 +21,13 @@
   <build_depend version_gte="0.3.0">python_qt_binding</build_depend>
   <build_depend>qt5-qmake</build_depend>
   <build_depend>qtbase5-dev</build_depend>
-  <build_depend>tinyxml_vendor</build_depend>
-  <build_depend>tinyxml</build_depend>
+  <build_depend>tinyxml2_vendor</build_depend>
+  <build_depend>tinyxml2</build_depend>
```

Comment:
I don't think there is a ROS package named `tinyxml2`?

Same below.

Reply:
Removed